Tasks/VsTest/vstest.ts

```diff
@@ -363,8 +363,12 @@ function executeVstest(testResultsDirectory: string, parallelRunSettingsFile: st
     tl.cd(workingDirectory);
     const ignoreTestFailures = vstestConfig.ignoreVstestFailure && vstestConfig.ignoreVstestFailure.toLowerCase() === 'true';
 
+    const envVars: { [key: string]: string; } = process.env;
+    utils.Helper.addToProcessEnvVars(envVars, 'TESTCASE_ACCESSTOKEN', tl.getVariable('TESTCASE_ACCESSTOKEN'));
```

Comment:
please fix the variable

Reply:
changed

---
In reply to: [161522719](https://github.com/Microsoft/vsts-tasks/pull/6224#discussion_r161522719) [](ancestors = 161522719)